lib/breadcrumbs.js

```diff
@@ -87,21 +87,22 @@ var wrappers = {
 
       // Don't capture breadcrumb for our own requests
       if (!Raven.dsn || url.indexOf(Raven.dsn.host) === -1) {
-        self.once('response', function (response) {
-          Raven.captureBreadcrumb({
-            type: 'http',
-            category: 'http',
-            data: {
-              method: method,
-              url: url,
-              status_code: response.statusCode
+        fill(self, 'emit', function (origEmit) {
+          return function (evt, maybeResp) {
+            if (evt === 'response') {
+              Raven.captureBreadcrumb({
+                type: 'http',
+                category: 'http',
+                data: {
+                  method: method,
+                  url: url,
+                  status_code: maybeResp.statusCode
+                }
+              });
             }
-          });
-
-          // No-op to consume response data since we added a response handler
-          // see https://nodejs.org/api/http.html#http_class_http_clientrequest
-          response.on('data', function () {});
```

Comment:
This `data` listener was causing `_readableState.flowing` to be true; see https://nodejs.org/api/stream.html#stream_three_states